lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -138,6 +138,11 @@ def debug_message
         else
           sorted_logs = container_logs.sort_by { |_, log_lines| log_lines.length }
           sorted_logs.each do |identifier, log_lines|
+            if log_lines.empty?
+              helpful_info << "  - Logs from container '#{identifier}': #{DEBUG_RESOURCE_NOT_FOUND_MESSAGE}"
```

Comment:
This is for consistency. However, in the interests of reducing noise, I'm wondering if we should go the other way and get rid of these messages entirely (don't output anything if logs/events can't be fetched. WDYT?